Gemfile

```diff
@@ -92,6 +92,11 @@ gem "jquery-ui-rails",   "5.0.5"
 gem "js_image_paths",    "0.1.0"
 gem "js-routes",         "1.2.9"
 
+# ES6
+gem "sprockets",            ">=3.0.0"
+gem "sprockets-es6",        "0.9.2"
+gem "babel-transpiler",     "0.7.0"
```

Comment:
I just followed a tutorial. This can probably be removed, then.
But, from what I've read, ES6 is fully supported by sprocket 4. So we wouldn't need `sprockets-es6` at all. The downside is that Sprockets 4 seems not backward compatible.